From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 16 Aug 2019 14:43:20 -0700
Subject: ACPICA: Debugger: remove redundant assignment on obj_desc
Patch-mainline: v5.4-rc1
Git-commit: 189c7213f4c5decddfe5969e28a1ce0bcae28cb1
References: jsc#SLE-16407

ACPICA commit f530f1acb3128136ad97c715fdaebbbeff283ee2

Pointer obj_desc is being initialized with a value that is never
read and it is being updated later with a new value. The initialization
is redundant and can be removed.

Addresses-Coverity: ("Unused value")

Link: https://github.com/acpica/acpica/commit/f530f1ac
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Bob Moore <robert.moore@intel.com>
Signed-off-by: Erik Schmauss <erik.schmauss@intel.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/acpica/dbobject.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/acpi/acpica/dbobject.c
+++ b/drivers/acpi/acpica/dbobject.c
@@ -394,7 +394,6 @@ void acpi_db_decode_locals(struct acpi_w
 	u8 display_locals = FALSE;
 
 	node = walk_state->method_node;
-	obj_desc = walk_state->method_desc;
 
 	/* There are no locals for the module-level code case */
 
